label_studio/static/samples/lorem_ipsum.txt

```diff
@@ -0,0 +1,4 @@
+Lorem ipsum dolor sit amet,
```

Comment:
Shall we put this into test/samples instead of runtime static dir? Or there is an intention to use this sample afterwards?

Reply:
i dont know, i need a txt sample for test scenario - and find this location
later we will make more scenarios with more different file samples